packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useContext, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import * as KeyCode from 'keycode-js';
+import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
+import styles from './WorklistDataGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String representing an attribute which identifies the element (or elements) that labels the grid.
+   */
+  ariaLabelledby: PropTypes.string,
+  /**
+   * String that labels the grid for accessibility.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * String that will be used to identify the Grid. This value will be used as the id attribute of the overall grid container,
+   * and it will be used to prefix other id attributes used for internal componentry.
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * Data for columns. By default, columns will be presented in the order given.
+   */
+  columns: PropTypes.arrayOf(WorklistDataGridPropTypes.columnShape),
+  /**
+   * Data for content in the body of the Grid. Rows will be rendered in the order given.
+   */
+  rows: PropTypes.arrayOf(WorklistDataGridPropTypes.rowShape),
+  /**
+   * String that specifies the default width for columns in the grid. Any valid CSS width value is accepted.
+   * To override the default value, provide width for the column that needs to be overridden.
+   */
+  columnWidth: PropTypes.string,
+  /**
+   * String that specifies the column height. Any valid CSS height value is accepted.
+   */
+  columnHeight: PropTypes.string,
```

Comment:
Is this really the headerHeight?  Based of the discussion in the UX meeting yesterday, it looks like we will only have the column headers with a header style treatment.  In that case, this would end up just being the header height for the worklist, right?

Reply:
As discussed, we renamed this to columnHeaderHeight.